app/jobs/hyku_addons/reindex_available_works_job.rb

```diff
@@ -1,8 +1,16 @@
 # frozen_string_literal: true
+
+# call with
+# HykuAddons::ReindexAvailableWorksJob.perform_now(['repo.hyku.docker'], cname_doi_mint: ["repo.hyku.docker"] )
+#
+# HykuAddons::ReindexAvailableWorksJob.perform_now(['repo.hyku.docker'])
+
 module HykuAddons
   class ReindexAvailableWorksJob < ApplicationJob
     # cnames is an array
-    def perform(cnames)
+    def perform(cnames, cname_doi_mint: [])
```

Comment:
Because like time we spoke last time, the test was testing perform and I will refactor to test the parameters passed in but even it that it does not give me any confidence that testing that the right parameters were passed means the job will function as expected.
This is why i have not updated the test.

Reply:
Insofar as all this job does is spawn other jobs, the only thing I would expect to see in an updated test is that the spawned jobs have the new parameter passed through.  That's not the case with `ReindexModelJob`.